index.js

```diff
@@ -82,7 +117,7 @@ function createIsUnavailable(elementDocument) {
   // "off" state, so we need to recursively check parents.
 
   function isOff(node, nodeComputedStyle) {
-    if (node === elementDocument.documentElement) return false;
+    if (node === elementDocument.documentElement || node instanceof ShadowRoot) return false;
```

Comment:
Won't this addition throw a `SyntaxError` for any browsers in which `ShadowRoot` is not defined?